src/components/Menus/Sidebar/Sidebar.jsx

```diff
@@ -98,7 +98,9 @@ const Sidebar = ({ options = [] }) => {
   removeOption("images", projectImages);
   removeOption("instructions", instructionsSteps);
 
-  const [option, setOption] = useState(isMobile ? "file" : null);
+  const [option, setOption] = useState(
+    instructionsSteps ? "instructions" : "file",
+  );
```

Comment:
So are we always defaulting the sidebar to open, even on mobile when there's no instructions?

Reply:
I've been searching through Slack but might need to @maxelkins on this one but I'm pretty sure that's what we agreed 😅